public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -96,19 +113,14 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, he
       )}
       <table className={styles.infoWrap}>
         <tbody>
-          {(mode === TooltipDisplayMode.Multi || mode == null) &&
-            displayValues.map((v, i) => (
-              <tr key={`${i}/${rowIndex}`} className={i === columnIndex ? styles.highlight : ''}>
-                <th>{v[0]}:</th>
-                <td>{renderValue(v[2])}</td>
-              </tr>
-            ))}
-          {mode === TooltipDisplayMode.Single && columnIndex && (
-            <tr key={`${columnIndex}/${rowIndex}`}>
-              <th>{displayValues[columnIndex][0]}:</th>
-              <td>{renderValue(displayValues[columnIndex][2])}</td>
+          {displayValues.map((v, i) => (
+            <tr key={`${i}/${rowIndex}`}>
+              <div className={v.highlight ? styles.highlight : ''}>
```

Comment:
It looks like this little styling workaround is breaking the build.

 Warning: validateDOMNesting(...): <td> cannot appear as a child of <div>.
          at td
          at div
          at tr
          at tbody
          at table
          at div